src/webhook/group_validation.py

```diff
@@ -2,18 +2,26 @@
 import sys, traceback
 import json
 import os
+import prometheus_client
+from prometheus_client import Counter
```

Comment:
Do you need this since you import the whole package above?

Reply:
I did the inverse and it worked. deleting `import prometheus_client` and retaining `from prometheus_client import Counter`